sunpy/map/mapsequence.py

```diff
@@ -511,3 +511,24 @@ def all_meta(self):
         Return all the meta objects as a list.
         """
         return [m.meta for m in self.maps]
+
+    def save(self, filepath, filetype='auto', **kwargs):
+        """Saves the SunPy Map object to a file.
+
+        Currently SunPy can only save files in the FITS format. In the future
+        support will be added for saving to other formats.
+
+        Parameters
+        ----------
+        filepath : str
+            Location to save file to.
+            Each map from the sequence will be saved separately.
+        filetype : str
+            'auto' or any supported file extension.
```

Comment:
If we only support Fits, should this not default to fits for now?

Reply:
Yeah, I was thinking about that even for `GenericMap`

Reply:
Hmm, I assume auto just defaults to fits in the io.write function?

Reply:
No, it doesn't, we get a `ValueError: The filetype provided () is not supported`.

Reply:
Sorry, I mean that if you provide "auto" for save, it will always default to a fits file?

Reply:
The default value in this method is `auto` right? Even if 'auto' is provided the same error occurs.

Reply:
What errors? I am slightly confused.

Reply:
When `filename=auto` is provided to `save`, we get `ValueError: The filetype provided () is not supported.`

Reply:
I wonder if thats a bug. I am not sure what is going on.

Reply:
Ah, so this occurs if we don't specify `filepath` with an extension. 
```python
map.save(filepath = "/home/map.fits", filetype='auto') # works fine.
```
```python
map.save(filepath = "/home/map", filetype='auto') # bugs out
```
The fault is in `sunpy.io.file_tools`
`filetype = pathlib.Path(fname).suffix[1:]`
Here, `filetype` is an empty string if no extension ("/home/map") is provided and this  raises the ValueError

Reply:
I see, leave it then.

Reply:
So, that works as intended right?

Reply:
Yeah